AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -45,6 +45,8 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
                                 "zh-CN"
                                ]
     let transcribeRegions = [
+                                "auto",
+                                "",
```

Comment:
Out of curiosity, what is the behavior of "not specified" for end user here?

Reply:
"Not specified" will use the same Chime meeting media region as Transcribe region. But Transcribe isn't available in all media regions (e.g., BOM) in which case the request will fail.